curriculum/challenges/english/blocks/workshop-recipe-tracker/66fbcdfed6848e48947dbbc9.md

```diff
@@ -16,13 +16,13 @@ Start by creating an empty array named `recipes`. This is the array you'll push
 You should create an array named `recipes`.
```

Comment:
The `recipes` array should be removed. It was removed in the issue because it wasn't really being utilized in the original. It would be different if this were an arrays workshop, and campers were learning and practicing about different array methods for the first time. But this is an objects workshop. The original project only have them create a `recipes` array and then push objects to it. But then nothing else happened. So the project needs to focus on the real learning goals which are practicing with object literals 👍🏾